blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx

```diff
@@ -0,0 +1,42 @@
+
+---
+authors: dr-farr
+title: 🕸 Top 5 social media regex for web developers 2022
+description: 
+slug: top-5-social-media-regex-for-web-developers-2022
+tags: [regex]
+---
+
+<!--truncate-->
```

Comment:
something I've just noticed on mine is that every used above this truncate line is used on this page as a preview - https://ventureharbour.tech/blog

Might be worth shifting the first paragraph up.